libsolidity/formal/CHC.cpp

```diff
@@ -119,10 +119,13 @@ void CHC::endVisit(ContractDefinition const& _contract)
 	if (!shouldVisit(_contract))
 		return;
 
-	auto errorAppl = (*m_errorPredicate)({});
-	for (auto const& target: m_verificationTargets)
+	for (unsigned i = 0; i < m_verificationTargets.size(); ++i)
+	{
+		auto const& target = m_verificationTargets.at(i);
+		auto errorAppl = error(i + 1);
```

Comment:
What does `Appl` stand for and why create a new local variable?

Reply:
It stands for an application of the error predicate.
It can also go into `query` directly